docs/reference/mapping/types/token-count.asciidoc

```diff
@@ -50,9 +50,10 @@ GET my_index/_search
 
 [NOTE]
 ===================================================================
-Technically the `token_count` type sums position increments rather than
+Technically, the `token_count` type sums position increments rather than
 counting tokens. This means that even if the analyzer filters out stop
-words they are included in the count.
+words they are included in the count.  
+If you don't want to count stop words, set option `enable_position_increments` to `false`
```

Comment:
You're right.
I see 2 options : 
- remove the note.
- only keep this part to just explain explain the mechanism :

> Technically, the `token_count` type sums position increments rather than counting tokens.

Which do you prefer ?

Moreover, I can add this precision to `enable_position_increments` definition :

> Set to `true` if you want to count all tokens computed by analyzer tokenizer. 

What do you think ?